### PR TITLE
Add bu.edu

### PR DESCRIPTION
**Pull request by seekheart**
Add Boston University

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -34,6 +34,16 @@ websites:
...
 
+    - name: Boston University
+      url: https://www.bu.edu
+      img: bu.png
+      tfa: Yes
+      sms: yes
+      phone: yes
+      software: yes
+      hardware: yes
+      doc: http://www.bu.edu/tech/support/duo/
```
Comment (kmpoppe) [inline _data/education.yml]:
```suggestion
      doc: https://www.bu.edu/tech/support/duo/
```
Please always use https-links if they are available (like in this case).

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Hello @seekheart and thank you for your contribution!
Would you please change the documentation link to https as I have shown in the code suggestion, thanks!
// Kai

File: _data/education.yml (hunk 1 above)
Comment (seekheart) [reply _data/education.yml]: done

**Comment by seekheart** [comment]
Hey @kmpoppe all done boss 😄

**Review by kmpoppe — APPROVED** [review]

**Comment by seekheart** [comment]
@kmpoppe what's the turn around time for merging?

**Comment by kmpoppe** [comment]
@seekheart We try to keep turn around to a minimum, yet all collaborators are doing this in their spare time as a hobby, nobody is working full-time on this project. Thus, it can take a few weeks until two people found the time to check everything.
Rest assured, we value every contribution!
// Kai

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**